cloudconfig/userdatacfg.go

```diff
@@ -41,6 +41,10 @@ type UserdataConfig interface {
 	// ConfigureJuju updates the provided cloudinit.Config with configuration
 	// to initialise a Juju machine agent.
 	ConfigureJuju() error
+	// ConfigureCustomOverrides updates the provided cloudinit.Config with
+	// user provided cloudinit data.  Data provided will overwrite current
+	// values with two exceptions. packages will be appended and runcmd.
```

Comment:
fix english here
s/p/P
Need to also say what is done with runcmd

Reply:
done